src/commands/create.ts

```diff
@@ -135,7 +129,7 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     const link = getLink(chain, res.meta.txn?.transactionHash as string);
     const out = { ...res, link, ensNameRegistered: false };
 
-    // TODO: I'm not sure how `check` would be false and statements.length < 2
+    // TODO: I'm not sure how `check` would be false and statements.length > 2
```

Comment:
This is a typo left over from a previous commit.

Reply:
Should we just remove it?

Reply:
Good call

Reply:
I removed the comment, but need another 👍